Formula/a/apitrace.rb

```diff
@@ -0,0 +1,133 @@
+    # license "BSD-3-Clause"
+  end
+  resource "thirdparty/zlib" do
+    url "https://github.com/madler/zlib/archive/21767c654d31d2dccdde4330529775c6c5fd5389.zip"
+    sha256 "b860a877983100f28c7bcf2f3bb7abbca8833e7ce3af79edfda21358441435d3"
+    # license "Zlib"
+  end
+  resource "thirdparty/directxmath" do
+    url "https://github.com/microsoft/DirectXMath/archive/e95d84892ec894967c48151f9106bd4388e85e63.zip"
+    sha256 "98e4ff2d3bd9231155e203172a826e9946953c68e949308bcf0446d8e005e17a"
+    # license "MIT"
+  end
+
+  # Patch for https://github.com/apitrace/apitrace/issues/826
+  # A resource instead of a patch, because we have to apply it between 2 building steps
+  resource "cgltrace-patch" do
+    url "https://github.com/apitrace/apitrace/commit/76d5b175da88a7267d85b9f5deb85324cb919048.diff"
+    sha256 "48411ce0897a4bdf788ae5b7fe858da049ba83d921ddd12afe0a9d176afe2e1e"
+    # license "MIT"
+  end
+
+  # - Disable system brotli because it's incompatible with universal macOS binary
+  # - Fix a compile bug with mixed QString and QLatin1String
+  patch :DATA
+
+  def install
+
+    # We'll build some universal library for tracing
+    ENV.permit_arch_flags
+
+    # Install all submodules
+    resource("thirdparty/brotli").stage { (buildpath + 'thirdparty/brotli').install Dir["*"] }
+    resource("thirdparty/gtest").stage { (buildpath + 'thirdparty/gtest').install Dir["*"] }
+    resource("thirdparty/libbacktrace").stage { (buildpath + 'thirdparty/libbacktrace').install Dir["*"] }
+    resource("thirdparty/libpng").stage { (buildpath + 'thirdparty/libpng').install Dir["*"] }
+    resource("thirdparty/snappy").stage { (buildpath + 'thirdparty/snappy').install Dir["*"] }
+    resource("thirdparty/zlib").stage { (buildpath + 'thirdparty/zlib').install Dir["*"] }
+    resource("thirdparty/directxmath").stage { (buildpath + 'thirdparty/directxmath').install Dir["*"] }
+
+    # Build the UI natively
+    system "cmake", "-S", ".", "-B", "build", "-DENABLE_GUI=ON", "-DENABLE_QT6=ON", *std_cmake_args
+    system "cmake", "--build", "build"
+    system "cmake", "--install", "build"
+
+    # We have to rebuild some parts of apitrace with a patch for the macOS tracer
+    # The patch also disables those parts which do not work, so we are only merging the improved parts with the previous install
+    # We also enable universal binaries so we can trace in Rosetta apps
+    resource("cgltrace-patch").stage { system "patch", "-d", buildpath, "-i", (Dir.pwd + "/76d5b175da88a7267d85b9f5deb85324cb919048.diff") }
+    system "cmake", "-S", ".", "-B", "build", "-DENABLE_GUI=OFF", "-DCMAKE_OSX_ARCHITECTURES=arm64;x86_64", *std_cmake_args
+    system "cmake", "--build", "build"
+    system "cmake", "--install", "build"
+
+  end
+
+  test do
+    system "#{bin}/apitrace", "trace", "glewinfo"
+    system "#{bin}/glretrace", "glewinfo.trace"
+    system "#{bin}/gltrim", "glewinfo.trace"
+    system "#{bin}/qapitrace", "--version"
+  end
```

Comment:
This test fails. I don't fully understand why.

I have `depends_on "glew" => :test` to have a trivial OpenGL app (although it actually doesn't seem to do anything that apitrace records, so ideally I'd find a different one).
However, it looks like `glewinfo` is not in the `PATH`.

How can I locate `glewinfo` during the test?